ios/StripeTerminalReactNative.swift

```diff
@@ -961,4 +997,33 @@ class StripeTerminalReactNative: RCTEventEmitter, DiscoveryDelegate, BluetoothRe
         let result = Mappers.mapFromReaderDisplayMessage(displayMessage)
         sendEvent(withName: ReactNativeConstants.REQUEST_READER_DISPLAY_MESSAGE.rawValue, body: ["result": result])
     }
+
+    func terminal(_ terminal: Terminal, didChange offlineStatus: OfflineStatus) {
+        self.terminal = terminal
```

Comment:
do we need this?

Reply:
it's the function of the OfflineDelegate, the function name is too brief..

Reply:
as @billfinn-stripe mentioned we should be able to use `Terminal.shared` everywhere without storing a local instance of the SDK. We should be able to remote this and the assignments on L1008 and L1014